docs/policies.md

```diff
@@ -1,3 +1,54 @@
+# Basic Expectations
```

Comment:
"Basic" doesn't really provide any information. It seems like this is a reference to the number of hours per week / year we expect people to work. Can we make that explicit in the title and maybe move the Compensation section into Benefits (and make make it Compensation & Benefits?)